lib/web/mage/adminhtml/browser.js

```diff
@@ -19,7 +19,8 @@ define([
 ], function ($, wysiwyg, prompt, confirm, alert, _) {
     window.MediabrowserUtility = {
         windowId: 'modal_dialog_message',
-        modalLoaded: false,
```

Comment:
Please, re-use `modaLoaded` property, because removing it is backward-incompatible change.

Reply:
ok, but then we need create a separate PR to remove this line https://github.com/magento/magento2-page-builder/blob/1605bd01f19e4b503aa97a0e78e9e828a81db0b3/app/code/Magento/PageBuilder/view/adminhtml/web/js/form/element/image-uploader.js#L79 in PageBuilder Repository, because we have always overridden this property, and gallery will initialized second time

Reply:
done :heavy_check_mark:

Reply:
@omiroshnichenko @Nazar65 This class is not API, and we shouldn't follow backward compatibility here.

Reply:
Separate PR opened at PageBuilder repo. https://github.com/magento/magento2-page-builder/pull/449